src/adapter/http/src/data/ingest_handler.rs

```diff
@@ -53,11 +55,24 @@ struct IngestTaskArguments {
 // until their data was processed. We may still provide a "synchronous" version
 // of push for convenience that waits for passed data to be flushed as part of
 // some block.
+/// Push data ingestion
+#[utoipa::path(
+    post,
+    path = "/ingest",
+    params(RequestParams),
+    request_body = Vec<u8>,
+    responses((status = OK, body = ())),
+    tags = ["kamu"],
+    security(
+        (),
```

Comment:
As I understand, this means okay for anonymous access.
The access for ingests should not be anonymous.

Reply:
Updated, but to be clear - other than `path`s, these schemas have no effect on runtime logic. An error in schema or security section will only result in invalid documentation.